lib/serializer/cbor/aws_iot_serializer_tinycbor_encoder.c

```diff
@@ -169,25 +205,23 @@ static AwsIotSerializerError_t _openContainer( AwsIotSerializerEncoderObject_t *
                 break;
 
             case AWS_IOT_SERIALIZER_CONTAINER_STREAM:
-                /* no action is required. */
+                /* TODO: figure out what it means to open stream container. */
```

Comment:
As per CBOR standard, data can be either map array or a sequence of scalar values/containers ( which is a stream ). So in my opinion we shouldn't support opening a stream container within another container. Thoughts ?

Reply:
I agree. Let me enforce it.